docs/searchpredicates.txt

```diff
@@ -110,7 +110,14 @@ Geoshape.circle(37.97, 23.72, 50)
 Geoshape.box(37.97, 23.72, 38.97, 24.72)
 //WKT (requires JTS for polygon, linestring and multi-point/line/polygon geometries)
 Geoshape.fromWkt("POLYGON ((35.4 48.9, 35.6 48.9, 35.6 49.1, 35.4 49.1, 35.4 48.9))")
-
+//MultiPoint
+Geoshape.geoshape(Geoshape.getShapeFactory().multiPoint().pointXY(60.0, 60.0).pointXY(120.0, 60.0).build());
+//MultiLine
+Geoshape.geoshape(Geoshape.getShapeFactory().multiLineString().add(Geoshape.getShapeFactory().lineString().pointXY(59.0, 60.0).pointXY(61.0, 60.0)).add(Geoshape.getShapeFactory().lineString().pointXY(119.0, 60.0).pointXY(121.0, 60.0)).build());
```

Comment:
Split this and the others below into multiple lines so they fit without (too much) scrolling